README.md

```diff
@@ -14,33 +14,36 @@ Pull requests to FSharp are welcome.
 FSharp is only compatible with **Sublime Text 3**.
 
 
-#### General steps
-
-* Clone this repository to any folder outside of Sublime Text's *Data* folder
-* Edit files as needed
-* Publish the project using `make install` or `.bin/Build.ps1`
-* Restart Sublime Text
-* Run the tests via command palette: *FSharp: Run Tests*
-
-
 #### Building
 
-See below for platform-specific instructions.
-
+### Linux/Mac
 
-### Development environment - Linux/Mac
+```shell
+./build.sh
+```
 
-Copy *FSharp*'s content manually to *{Sublime Text Data Path}/Packages/FSharp*.
+### Windows
 
+```shell
+build.cmd
+```
 
-### Development environment - Windows
+The build target 'Install' will copy the built output into ST3's Packages directory:
 
-#### Requirements
+```shell
+./build.sh install
+```
 
+If you're running a portable install of Sublime Text you need to tell the build script where to install: 
 
-You must set `$STDataPath` in your PowerShell session to Sublime Text's *Data* path.
+```shell
+./build.sh install sublimeDir="d:\AppData\ST3"
```

Comment:
We're talking about Windows here, yet `build.sh` is used in the example. Mistake?